h/services/user_delete.py

```diff
@@ -87,7 +87,7 @@ def purge_deleted_users(self, limit=1000):
                 completed_jobs.append(job)
                 continue
 
-            log.info("Purging user: %s", user.userid)
+            log.info("Purging user: %r", user)
```

Comment:
Let's just always log the whole `User.__repr__()` not just the userid. That way we just need to get the `User.__repr__()` method right once and then it's always used whenever logging a user.